BadPotato1007.md

```diff
@@ -0,0 +1,65 @@
+
+
+This project will be opensource and will have instructions on how to make it on instructables, so that other hobbiests like me can make it too.
+
+Here are some pics that can give you an idea of what it will look like
+![image](https://user-images.githubusercontent.com/70943638/210223410-4a1b08bf-463a-4cfe-9a54-e88d3ab10a53.png)
+![image](https://user-images.githubusercontent.com/70943638/210223450-f925ec12-252a-4886-9e0f-745854d605a6.png)
+
+
+
+## Plan
+
+1. To start off, I will need a new rasberryPi device as my old one shortcircuited and the gpio pins no longer work
+2. Then I will have to connect all the hardware to the rasberryPi, which includes 
+    1. The 7" touch display 
+    2. The 16 capacitive switches as the buttons for the drum pad with the help of a port expander [MCP23017]
+    3. 6 Output Shift registers to control the 16 Rgb leds for the drum pad [74HC595]
+    4. The mic              [These will be connected to the rasberryPi via a usb soundcard]
+    5. The mono speaker     [These will be connected to the rasberryPi via a usb soundcard]
+    6. The 2 rotary encoders [Change modes and select drum sounds]
+    7. The 2 potentiometers [Volume]
+3. Then I will have to write the code [Planning to use python]
+4. Make a nice good professional looking case with the help black acrylic sheets and some 3d printed parts
+5. Use the drumbox looper and make the project opensource with instructions on how to make it on instructables
+
+
+## Budget
+
+
+| Product                   | Supplier/Link                                                                                      | Cost   |
+| ---------------           | -------------------------------------------------------------------------------------------------- | ------ |
+| 7" LCD display            | https://www.amazon.in/REES52-Display-1024X600-Function-Raspberry/dp/B07GDLB7TM/product/1957        | $81.25 |
+| Rotry Encoder             | https://robu.in/product/m274-360-degree-rotary-encoder-module-brick-sensor/                        | $0.68  |
+| Rasberry pi 3b            | https://www.olx.in/item/raspberry-pi-3b-6-pieces-brand-new-condition-iid-1702194889                | $68.75 |
+| MCP23017                  | https://robu.in/product/mcp23017-16-bit-input-output-expander-with-i2c-interface-ic-dip-28-package | $4.00  |
+| 74HC595                   | https://robu.in/product/sn74hc595n-dip16/  (owned)                                                 | $1.75  |
+| Potentiometer             | https://robu.in/product/ptv09a-4225f-b103-rotary-potentiometer/                                    | $1.30  |
+| Capacitive Sensor         | https://robu.in/product/ttp223-touch-key-module-2pcs/                                              | $6.00  |
+| 12V 2A Adapter            | Owned                                                                                              | $3.00  |
```

Comment:
for items that you own, could you please make the cost $0? thank you sm :)

Reply:
Done!